src/Builder/DatabaseBuilder.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+namespace FiveamCode\LaravelNotionApi\Builder;
+
+use FiveamCode\LaravelNotionApi\Endpoints\Databases;
+use FiveamCode\LaravelNotionApi\Entities\Database;
+use Illuminate\Support\Collection;
+
+/**
+ * Class DatabaseBuilder.
+ */
+class DatabaseBuilder
+{
+    /**
+     * @var array
+     */
+    private array $payload;
+
+    /**
+     * DatabaseBuilder constructor.
+     *
+     * @param  Databases  $databasesEndpoint
+     */
+    public function __construct(private Databases $databasesEndpoint)
```

Comment:
If we don't use `$databasesEndpoint` in the constructor, can we define it next to payload up there?

Reply:
yes, done!